### ISSUE TITLE
Language server currently doesn't display when tables are created in the wrong order

### ISSUE THREAD

**Issue by FrancescoMasaia** [kwilteam/kuneiform-ls#10]
When using VSCode, no errors appear with the extension enabled. However, when I execute the migration to the Kwil network, I encounter an error stating that a foreign key cannot be applied because the table doesn't exist yet. It would be extremely helpful to identify such errors earlier in the process. This is especially important for long files, as it is much more challenging to correct errors after the file is complete rather than addressing them during the writing phase.
Also foreign key type checking doesn't seem to work (I've tried it with uuid/text and it doesn't give any advice about that error)

``` sql
table a {
id uuid,
table_b_id uuid,
foreign key (table_b_id) references table_b(id) on delete cascade 
#this should give error
}

table b {
id uuid,
name text
}
```

**Comment by KwilLuke** [comment]
Hey @FrancescoMasaia, thanks for bringing this up! We will investigate and try to get back ASAP.

cc @brennanjl - it looks like this is may be a parser/validation issue.

**Comment by brennanjl** [comment]
Apologies @FrancescoMasaia I did not have notifications turned on for this repo. This appears to be a bug with the parsers error handling, I will get it addressed asap.

**Comment by brennanjl** [comment]
@charithabandi can you look into this? I am unable to replicate, AFAICT this seems to be handled properly.

**Comment by brennanjl** [comment]
Fix is here @charithabandi: https://github.com/kwilteam/kwil-db/pull/935

**Comment by charithabandi** [comment]
![Screenshot 2024-09-19 at 11 47 47 AM](https://github.com/user-attachments/assets/d6b63964-b31a-4058-9608-f91baaae6293)
This issue should be fixed on v0.0.5, please update and restart your extensions.
